docs/tutorials/hyp3_insar_stack_for_ts_analysis.ipynb

```diff
@@ -47,22 +47,22 @@
     "\n",
     "To run this notebook, you'll need a conda environment with the required dependencies. You can set up a new environment (recommended) and run the jupyter server like:\n",
     "```shell\n",
-    "conda create -n hyp3-mintpy python=3.8 asf_search hyp3_sdk \"mintpy>=1.3.2\" pandas jupyter ipympl\n",
+    "conda create -n hyp3-mintpy python=3.10 asf_search hyp3_sdk \"mintpy>=1.3.2\" pandas jupyter ipympl\n",
```

Comment:
No concerns with updating to `python=3.10`, but I'm curious, did you have trouble using `python=3.8`? I was able to create the environment using 3.8 without issue.

Reply:
I have trouble with python=3.8, When I create the hyp3-mintpy with python=3.8, I stuck at code import asf_search as asf, it complains that I can not load the module.